### PR TITLE
Fix positions new

### PR DESCRIPTION
**Pull request by zdenkas**
Three changes in hg38 for Missense protein variant types pathogenic sheet

### CODE REVIEW

File: notebooks/00020_fix_positions.ipynb [hunk 1]
```diff
@@ -0,0 +1,2090 @@
+{
```
Comment (zdenkas) [inline notebooks/00020_fix_positions.ipynb]: <p>@dianagroch can you please check this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1474856567'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='31'/>

Comment (zdenkas) [inline notebooks/00020_fix_positions.ipynb]: <p>@dianagroch can you please check this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1474856569'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (zdenkas) [inline notebooks/00020_fix_positions.ipynb]: <p>@dianagroch can you please check this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1474856572'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

File: notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb [hunk 2]
```diff
@@ -0,0 +1,2090 @@
+{
```
Comment (dianagroch) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1]: <p>agree</p><h2>rs878853654</h2><p>ACMG likely pathogenic</p><p>ClinVar 1x likely pathogenic</p><p>gnomAD v.4 ENF zero</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1478028368'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='31'/>

Comment (dianagroch) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1]: <p>agree</p><p>in position X:154413499 (hg38) is A</p><p>c.302A&gt;T</p><p>p.Asp101Val</p><p>ACMG VOUS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1478057620'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (dianagroch) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1]: <p>agree</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/#comment-1478057622'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='42'/>

**Pull request merged**